src/libraries/Microsoft.Extensions.DependencyModel/src/DependencyContextWriter.cs

```diff
@@ -59,10 +59,10 @@ private static void WriteRuntimeTargetInfo(DependencyContext context, Utf8JsonWr
         private static void WriteRuntimeGraph(DependencyContext context, Utf8JsonWriter jsonWriter)
         {
             jsonWriter.WriteStartObject(DependencyContextStrings.RuntimesPropertyName);
-            foreach (RuntimeFallbacks runtimeFallback in context.RuntimeGraph)
+            foreach (RuntimeFallbacks runtimeFallback in context.RuntimeGraph.OrderBy(r => r.Runtime, StringComparer.Ordinal))
```

Comment:
Is there a risk of this breaking how the runtime fallback is computed/used?

Reply:
@ericstj / @ViktorHofer Can one of you weigh in on if this can affect behavior adversely?

Reply:
The host implements runtime lookup.  @elinor-fung I would expect that nothing here is order dependent since the host should be probing in the order defined in the `runtimes` section of the shared-framework's deps file.